pkg/controller/cluster/resource/resource.go

```diff
@@ -133,6 +133,17 @@ func StatefulSetForRack(r scyllav1alpha1.RackSpec, c *scyllav1alpha1.Cluster, si
 								EmptyDir: &corev1.EmptyDirVolumeSource{},
 							},
 						},
+						{
+							Name: "scylla-config-volume",
+							VolumeSource: corev1.VolumeSource{
+								ConfigMap: &corev1.ConfigMapVolumeSource{
+									LocalObjectReference: corev1.LocalObjectReference{
+										Name: "scylla-config",
```

Comment:
Do we want to hardcode a specific name here?
What if the users wants to have multiple ConfigMaps for different Scylla Clusters?
Is that a use-case we would support?

My initial thought here was to make this a field in the RackSpec.
We can save it for a followup PR, if you feel that's better.

Reply:
I didn't like this when I wrote it but at the time it just worked. Putting it in the RackSpec seems like a good idea.

Reply:
I have pushed a new commit with the name configurable. Changing it does not work yet though.